lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -512,17 +456,20 @@ def extract_entry_labels(group_resource, record)
         end
       end
 
-      # If a field is present in the label_map, send its value as a label
-      # (mapping the field name to label name as specified in the config)
-      # and do not send that field as part of the payload.
+      # If the name of a field in the record is present in the @label_map
+      # configured by users, report its value as a label and do not send that
+      # field as part of the payload.
       common_labels.merge!(fields_to_labels(record, @label_map))
 
+      # Cloud Functions.
+      # TODO TODOTODO
```

Comment:
Oh, this is a reminder for myself to double check the influence and move this part to be right after the Cloud Functions tag parsing above. Completed the TODO.